### DISCUSSION TITLE
Jadikan (Mediatek Helio P35) Dapat di mainkan,

### DISCUSSION THREAD

**Discussion by ikyyRisky** [skyline-emu/skyline#1562] [Q&A]
Saat saya ingin bermain, semua gamenya akan force close apakah kedepannya akan menjadi lebih baik?

**Comment by ihsbleh13** [comment]
> Saat saya ingin bermain, semua gamenya akan force close apakah kedepannya akan menjadi lebih baik?

Pake bahasa inggris disini rata2 orang luar apalagi devnya

**Comment by Adt67** [comment]
P30 terlalu rendah,minimal setara snapdragon 750
